src/restic/node.go

```diff
@@ -496,9 +568,35 @@ func (node *Node) fillExtra(path string, fi os.FileInfo) error {
 		err = errors.Errorf("invalid node type %q", node.Type)
 	}
 
+	switch runtime.GOOS {
+	case "windows", "openbsd":
+		return err
+	}
+	if err = node.fillAcls(path); err != nil {
+		return err
+	}
+
 	return err
 }
 
+func (node *Node) fillAcls(path string) error {
+	xattrs, e := Listxattr(path)
+	// ignore paths for which no acl can be obtained silently
+	if e == nil {
```

Comment:
Why is this error ignored? When the attributes for a file cannot be listed, isn't that an error which should at least be displayed to the user?